fast/stages/02-networking/variables.tf

```diff
@@ -25,17 +25,15 @@ variable "custom_adv" {
   type        = map(string)
   default = {
     cloud_dns             = "35.199.192.0/19"
+    gcp_all               = "10.128.0.0/16"
+    gcp_dev               = "10.128.32.0/19"
+    gcp_landing           = "10.128.0.0/19"
```

Comment:
Nit: this implies a contiguous IP space across different regions, which is not a given.
Not gating as it's simple enough to modify for customers.